ddtrace/example_test.go

```diff
@@ -45,6 +45,7 @@ func Example_opentracing() {
 	// Start a Datadog tracer, optionally providing a set of options,
 	// returning an opentracing.Tracer which wraps it.
 	t := opentracer.New(tracer.WithAgentAddr("host:port"))
+	defer tracer.Stop() // important for data integrity (flushes any leftovers)
```

Comment:
I'd write this as `// Flushes any in-flight data.`

Reply:
You're right. "Leftovers" is not the most fantastic nor technically correct word.